nwb_conversion_tools/datainterfaces/icephys/baseicephysinterface.py

```diff
@@ -0,0 +1,145 @@
+"""Author: Luiz Tauffer."""
+from abc import ABC
+from typing import Optional
+
+from pynwb import NWBFile
+from pynwb.device import Device
+from pynwb.icephys import IntracellularElectrode
+
+from ...basedatainterface import BaseDataInterface
+from ...tools.neo import (
+    get_number_of_electrodes,
+    get_number_of_segments,
+    write_neo_to_nwb,
+)
+from ...utils import (
+    OptionalFilePathType,
+    get_schema_from_hdmf_class,
+    get_schema_from_method_signature,
+    get_base_schema,
+)
+
+
+try:
+    from ndx_dandi_icephys import DandiIcephysMetadata
+
+    HAVE_NDX_DANDI_ICEPHYS = True
+except ImportError:
+    HAVE_NDX_DANDI_ICEPHYS = False
+
+
+class BaseIcephysInterface(BaseDataInterface, ABC):
+    """Primary class for all intracellular NeoInterfaces."""
+
+    neo_class = None
+
+    @classmethod
+    def get_source_schema(cls):
+        source_schema = get_schema_from_method_signature(class_method=cls.__init__, exclude=[])
+        return source_schema
```

Comment:
This is almost the same as the parent code: https://github.com/catalystneuro/nwb-conversion-tools/blob/main/nwb_conversion_tools/basedatainterface.py#L10-L13

Is there a reason you need to override that we should be aware of?